### ISSUE TITLE
Make `use_turbulence_correction=True` the default value for get_farm_power()

### ISSUE THREAD

**Issue by zerweck** [NREL/floris#173]
**Is your feature request related to a problem? Please describe.**
`get_farm_power()` is used many times throughout the library when AEP Calculations happen
However, turbulence correction via `use_turbulence_correction` is never passed as a parameter from the calling method, and therefore all AEP calculations default to ignoring turbulence effects (`use_turbulence_correction=False`)

**Describe the solution you'd like**
I wonder what the reason for this default value is, since as a user I want to utilize the maximum simulation accuracy when calculating the AEP - for example for layout optimization.

**Describe alternatives you've considered**
Set the default value to true, or at least give the user of functions such as `get_farm_AEP` or `_AEP_loop_wd` the transparency that turbulence influence is ignored here.

**Comment by paulf81** [comment]
@rafmudaf , given the policy in #715 , propose we close this issue (turbulence correction will be re-included in 4.0 as well)

**Comment by misi9170** [comment]
Agreed with the comment above, closing as v2 is not under active development.
